models/Review.js

```diff
@@ -43,6 +45,15 @@ Review.pre('save', function(next) {
 		}
 	}
 
+	if (this.reply !== undefined) {
+
+		let length = this.reply.text.length;
+
+		if (length > 1000) {
+			this.reply = this.reply.text.substring(0, 999);
```

Comment:
Hmm, not cool for users I think, there should be a margin, or something like the Twitter counter on the client side.